### PR TITLE
8/14/2022 - Inara Compatibility Update

### PR DESCRIPTION
**Pull request by TuggyBoat**
- Changed request format to scrape new website correctly
- Moved Inara defs to separate files for ease of use/access
- Optimized imports in main file

### CODE REVIEW

**Comment by TuggyBoat** [comment]
I did this in 3.10, but it should still work in 3.9

File: bot.py [hunk 1]
```diff
@@ -9,20 +9,19 @@
...
-from datetime import datetime
 import traceback
+from datetime import datetime
+
+import discord
+from discord.ext import commands, tasks
+from dotenv import find_dotenv, load_dotenv, set_key
+from texttable import Texttable
+
+from inara_requests import *
```
Comment (gcruicks) [inline bot.py:24]: Better to explicitly import what you need than Import * which can lead to unintended side effects and hidden things later on

Comment (durzo) [reply bot.py:24]: `import *` doesnt play well with vscode too, this must be explicit.

Comment (durzo) [reply bot.py:24]: @TuggyBoat what is the reason behind moving just these 2 functions into their own module?

**Comment by durzo** [comment]
These fixes have been pulled in via another commit, thank you.

**Pull request closed without merging**